scratch-vm/src/extension-support/tw-iframe-extension-worker.js

```
@@ -1,5 +1,14 @@
 const uid = require('../util/uid');
-const frameSource = require('./tw-load-script-as-plain-text!./tw-iframe-extension-worker-entry');
+
+// Bob the Builder
+// Can we fix it?
+// Bob the Builder
+// Yes, we can!
+const getFrameSource = () => {
+    const publicPath = typeof __webpack_public_path__ === 'string' ? __webpack_public_path__ : '';
```

ESLint

[error] '__webpack_public_path__' is not defined.

(no-undef)